pipelines/build/regeneration/build_pipeline_generator.groovy

```diff
@@ -60,6 +60,7 @@ node('master') {
     config.JOB_NAME = "weekly-openjdk${javaVersion}-pipeline"
     config.SCRIPT   = "pipelines/build/common/weekly_release_pipeline.groovy"
     config.PIPELINE = "openjdk${javaVersion}-pipeline"
+    config.weekly_release_scmReferences = target.weekly_release_scmReferences
```

Comment:
The variables above a set globally (if we're following CAPITAL convention) - why not this one?

Reply:
yes, I was following the convention of targetConfigurations.. I saw these as well